RockLib.Diagnostics/Tracing.cs

```diff
@@ -13,7 +13,10 @@ namespace RockLib.Diagnostics
     /// Defines methods for configuring the <see cref="Trace"/> static class and retrieving
     /// <see cref="TraceSource"/> objects.
     /// </summary>
+#pragma warning disable CA1724
```

Comment:
[CA1724: Type names should not match namespaces](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1724) - Suggested to suppress for already existing libraries.